Assets/Scripts/Player.cs

```diff
@@ -62,8 +115,10 @@ void Start ()
 		crosshairTimer = crosshair.transform.Find("CrosshairTimer").GetComponent<Image>();
 	}
 
-	void Update ()
+	void Update()
 	{
+		ConnectedDevices.DetectDevices();
```

Comment:
Good idea to put it in Update, but measure and let me know how much time this takes. May be a better idea to check this every `x`th frame